.golangci.yaml

```diff
@@ -81,7 +81,6 @@ linters:
     - nilerr  # finds the code that returns nil even if it checks that the error is not nil
     - nolintlint  # reports ill-formed or insufficient nolint directives
     - nosprintfhostport  # checks for misuse of Sprintf to construct a host with port in a URL
-    - perfsprint  # Golang linter for performance, aiming at usages of fmt.Sprintf which have faster alternatives
```

Comment:
Zie je een issue met deze linter?

Reply:
ja ik vond het belangrijker/mooier/overzichtelijker om alles consistent te doen, ook als er geen variabelen zijn, dan te mixen. de performance winst lijkt me wel mee vallen.

mixen:
errors met fmt.errorf vs errors.new
string met fmt.sprintf("foo") gewoon "foo"

@kad-korpem join the discussion :)

Reply:
Hmm, ja. Voor GoKoala heb ik het uiteindelijk gewoon wel volgens advies van de linter opgelost. Het heeft wat impact op consistentie, maar het is uiteindelijk ook niet echt logisch om een string formatter te gebruiken als je een normale string gebruikt.
Enige recommendation van `perfsprint` die ik niet fantastisch mooi vind, is om iets als `greeting := fmt.Sprintf("hello, %s", name)` te vervangen door `greeting := "hello, " + name`. Maar als dat (in ieder geval in theorie) net zo goed of beter performt, kan ik er wel mee leven.

Reply:
@rkettelerij hak jij de knoop door? mijn mening is niet sterk

Reply:
Punt van consistentie snap ik wel, en is wel terecht. Aan de andere kant de linter helpt je wel snel om dit aan te wijzen (linter werkt ook in de IDE). Zou het erin laten, scheelt ook weer overal linter config aanpassen.